scripts/prebuild.js

```diff
@@ -132,11 +132,25 @@ const setVersion = async (buildType, tagName) => {
   fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, undefined, 2));
 };
 
+const setDependencies = () => {
+  const packageJsonPath = path.join(__dirname, "..", "cysync", "package.json");
+  const packageJson = JSON.parse(fs.readFileSync(packageJsonPath));
+
+  if (process.platform === "darwin") {
+    packageJson.resolutions = {
```

Comment:
This will override any existing resolutions present. Though we don't have any now, we may have some in the future.
Preserve any existing resolutions.

Reply:
Nice catch @irshadCypherock . Done the appropriate changes